.travis.yml

```diff
@@ -17,5 +17,8 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
-script:
+
+before_script:
```

Comment:
Why you rename this to `before_script`?

Reply:
To be honest, for no particular reason 😅 . When I added postgres service I replicated it from another project... I'll change it back to script anyways.

Reply:
Actually I guess I did it because I needed the migrations to run before, but as we don't need migrations... 👍